src/locales/en_US.js

```diff
@@ -719,6 +719,16 @@ const strings = {
   fio_get_requests_error: 'There was an issue fetching requests',
   fio_confirm_request_error: 'There was an issue during request send',
   fio_reject_request_error: 'There was an issue during request rejection',
+  fio_confirm_request_header: 'Send FIO Request',
+  fio_confirm_request_from: 'Send Request From',
+  fio_confirm_request_to: 'Send Request To',
+  fio_confirm_request_fio_title: 'Choose FIO Address',
+  fio_confirm_request_fio_subtitle_to: 'Enter address to request payment to',
```

Comment:
On Send Request From, use subtitle : Send request from which address?
On Send Request To, use subtitle : Enter address to request payment from